app/features/quizzify/document_loaders.py

```diff
@@ -0,0 +1,376 @@
+from langchain_community.document_loaders import YoutubeLoader
+from langchain_community.document_loaders import PyPDFLoader
+from langchain_community.document_loaders.csv_loader import CSVLoader
```

Comment:
You can import all the document loaders in one line for best practice, like: 
from langchain_community.document_loaders import TextLoader, UnstructuredURLLoader, UnstructuredPowerPointLoader, Docx2txtLoader, UnstructuredExcelLoader, UnstructuredXMLLoader

Reply:
Done!